lib/zstd.h

```diff
@@ -1160,6 +1162,11 @@ typedef enum {
                                  * Decoder cannot recognise automatically this format, requiring this instruction. */
 } ZSTD_format_e;
 
+typedef enum {
```

Comment:
Maybe just add a quick comment here that this is used with the `ZSTD_d_forceIgnoreChecksum` parameter?